### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by salvadorrueda**
Hi. Spanish translation is ready to merge. I also fixed some minor issues with values in "untranslatable" entries. 

Let me know if everything is ok. Have fun.

### CODE REVIEW

**Comment by salvadorrueda** [comment]
Ups. What happened?

File: app/src/main/res/values-es-rES/strings.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pick_file">Elige un fichero</string>
+    <string name="toast_pick_file_error">Imposible acceder al fichero. Comprueba el estado del gestor de ficheros.</string>
```
Comment (elJoa) [inline app/src/main/res/values-es-rES/strings.xml:4]:
```suggestion
    <string name="toast_pick_file_error">Fue imposible abrir el fichero. Comprueba el estado del gestor de ficheros.</string>
```

File: app/src/main/res/values-es-rES/strings.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <string name="toast_http_code_error">"Problema cargando página, http error code : "</string>
```
Comment (elJoa) [inline app/src/main/res/values-es-rES/strings.xml:5]:
```suggestion
    <string name="toast_http_code_error">"Ocurrió un problema cargando la página, código de error HTTP: "</string>
```

File: app/src/main/res/values-es-rES/strings.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <string name="toast_io_exception">No se puede cargar esta página for alguna razón.</string>
+    <string name="toast_ssl_error">Conexión segura fallida.</string>
+    <string name="action_about">Acerca de</string>
+    <string name="version">Versión</string>
+    <string name="intro">Repetir Intro</string>
```
Comment (elJoa) [inline app/src/main/res/values-es-rES/strings.xml:10]:
```suggestion
    <string name="intro">Repetir intro</string>
```

File: app/src/main/res/values-es-rES/strings.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <string name="email">E-mail</string>
+    <string name="description_open">Por supuesto esta es una aplicación de código abierto. Tienes la opción de hacer tus própias modificaciones. El enlace al código fuente está en la sección About. Acerca de</string>
```
Comment (elJoa) [inline app/src/main/res/values-es-rES/strings.xml:12]:
```suggestion
    <string name="description_open">Por supuesto esta es una aplicación de código abierto. Tienes la opción de hacer tus propias modificaciones. El enlace al código fuente está en la sección Acerca de.</string>
```

File: app/src/main/res/values-es-rES/strings.xml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    <string name="title_open">Libre</string>
+    <string name="description__intro">Un simple visualizador de documentos Pdf</string>
+    <string name="myLicense">Licencia</string>
+    <string name="privacy">Política de privacidad</string>
+    <string name="notice">Buenas notícias. Pdf Viewer Plus está disponible en F-Droid!</string>
```
Comment (elJoa) [inline app/src/main/res/values-es-rES/strings.xml:17]:
```suggestion
    <string name="notice">Buenas notícias. ¡Pdf Viewer Plus está disponible en F-Droid!</string>
```

File: app/src/main/res/values-es-rES/strings.xml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    <string name="pref_primary_color_title">Color principal</string>
+    <string name="pref_primary_color_summary">Visualizado con mayor frecuencia en la aplicación.</string>
+    <string name="pref_accent_color_title">Color de los títulos</string>
+    <string name="pref_accent_color_summary">Color de partes destacables de la interficie.</string>
+    <string name="pref_background_color_title">Color de fondo</string>
+    <string name="pref_background_color_summary">El color subyacente del contenido de la aplicación.</string>
+    <string name="pref_category_appearance">Aparencia</string>
+    <string name="pref_colored_navigation_bar_title">Color de la barra de navegación</string>
+    <string name="pref_colored_navigation_bar_summary">Aplica el color principal a la barra de navegación</string>
+    <string name="password">Password</string>
```
Comment (elJoa) [inline app/src/main/res/values-es-rES/strings.xml:43]:
```suggestion
    <string name="password">Contraseña</string>
```

File: app/src/main/res/values-es-rES/strings.xml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    <string name="quality">Renderización de alta calidad</string>
+    <string name="settings">Preferencias</string>
+    <string name="alias">Anti-Aliasing</string>
+    <string name="scroll">Desplazamiento horizontal</string>
+    <string name="fling">Page Fling</string>
+    <string name="qualitysettings">Calidad</string>
+    <string name="scrollsettings">Desplazamientos</string>
+    <string name="title_activity_settings">Preferencias</string>
+    <string name="reload">Recargar PDF</string>
+    <string name="reload_sum">Para poder aplicar los cambios, el PDF debe ser volver a abrir.</string>
```
Comment (elJoa) [inline app/src/main/res/values-es-rES/strings.xml:55]:
```suggestion
    <string name="reload_sum">Para poder aplicar los cambios, el PDF se debe volver a abrir.</string>
```

**Review by elJoa — CHANGES_REQUESTED** [review]
Hey! I found some errors in the translation. I think I left some so let me know! :D

**Comment by JavaCafe01** [comment]
Hello @elJoa and @salvadorrueda , I believe there are some updated strings, can you all take a look and update the translations? Please take a look in the dev branch and target the dev branch. Thanks!

**Comment by elJoa** [comment]
Sure, I'll take a look when I get to home.

**Comment by elJoa** [comment]
Ok, I have all strings translated. Should I open a separate pull request since I can't upload them here?

**Comment by JavaCafe01** [comment]
#90 merged

**Pull request closed without merging**